readme.md

```diff
@@ -17,7 +17,17 @@ Using jQuery, write code that makes an AJAX get request to "http://kittengifs.co
 
 Your Answer:
 ```js
-
+@.ajax({
+  type: 'GET',
+  dataType: 'json',
+  url: http://kittengifs.com/gifs,
+}).done(function(response){
+  var count = 0;
+  for (var i=0; i<response.length; i++) {
+    count = count+1;
+  }
+  console.log(count);
+});
```

Comment:
:+1: That works.  Is there a simpler way?

Hint: how did you know many times your loop should run?  ;)